whois-rdap/src/test/resources/IanaAdministrativeRangeTest.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I suggest downloading the real (full) XML as I don't expect it to ever change (also don't have a dependency on an external resource that may not be available and block startup) and commit to src/main/resources and also use that for testing. Add a link to the URL as a comment in the code.
Or, download once to the var/ directory and don't re-download if it already exists.